frontend/src/containers/AdminPanel/AdminPanel.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import Header from '../../components/Header/Header.js';
+import Footer from '../../components/Footer/Footer.js';
+import Recipes from '../../components/AdminControls/Recipes';
+import Users from '../../components/AdminControls/Users';
+import { TabContent, TabPane, Nav, NavItem, NavLink, Button } from 'reactstrap';
+import classnames from 'classnames';
+import './AdminPanel.css';
+
+class AdminPanel extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeTab: '1',
```

Comment:
What do you think of using more descriptive tab names, like `members` and `recipes`?